### ISSUE TITLE
Invalid CSS is generated for `{screen}:display` rules

### ISSUE THREAD

**Issue by jeisses** [green-coder/girouette#65]
First of all, thanks a lot for making this library! It's very convenient and fun to use

I noticed an issue when using classes for responsive display properties, such as `sm:block`, `md:table-cell`, etc.

For example, `sm:flex` generates the following CSS:

```
@media (min-width: 640px) {

  .sm\:flex {
    display: sm:flex;
  }

}
```

Which should be:
```
@media (min-width: 640px) {

  .sm\:flex {
    display: flex;
  }

}
```

**Comment by green-coder** [comment]
Thank you for reporting the issue.

**Comment by green-coder** [comment]
@jeisses This bug seems to be a duplicate of #55 which was fixed in v0.0.3 and which has a test in the codebase already: 
https://github.com/green-coder/girouette/blob/46ca014ebdc95a9edbcc46ae4d1f00d261bf683a/lib/girouette/test/girouette/tw/layout_test.cljc#L28-L32

Which version do you use?

**Comment by green-coder** [comment]
I also tested `sm:block` and `md:table-cell`, they all work as intended.

Closing as invalid.
